BaseFilter/sections/adservers.txt

```diff
@@ -895,6 +895,9 @@
 ! To fix the issue with the CNAME trackers, add problematic domains here:
 ! https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/exclusions.txt
 !
+! GoAdserver : go.goadserver.com disguised advertising networks
```

Comment:
not ad servers - tracking
https://github.com/AdguardTeam/AdguardFilters/blob/5ebe28dced37f1866128db157527724f4f8f93cb/SpywareFilter/sections/tracking_servers.txt#L72